Use regular `-I` to include BLAS header files from EasyBuild dependencies.
author: Alex Domingo (Vrije Universiteit Brussel)
--- ggml/src/ggml-blas/CMakeLists.txt.orig	2026-09-22 17:18:40.943083000 +0200
+++ ggml/src/ggml-blas/CMakeLists.txt	2026-09-22 17:18:53.925387000 +0200
@@ -93,7 +93,7 @@
     endif()
 
     target_link_libraries     (ggml-blas PRIVATE ${BLAS_LIBRARIES})
-    target_include_directories(ggml-blas SYSTEM PRIVATE ${BLAS_INCLUDE_DIRS})
+    target_include_directories(ggml-blas PRIVATE ${BLAS_INCLUDE_DIRS})
 else()
     message(FATAL_ERROR "BLAS not found, please refer to "
                         "https://cmake.org/cmake/help/latest/module/FindBLAS.html#blas-lapack-vendors"
